v0.7.0/concepts/mev-commit-chain/bridging-details.mdx

```diff
@@ -21,7 +21,7 @@ There are inherent security assumptions in bridging ETH to the mev-commit chain.
 - POA signers that maintain mev-commit chain state.
 - Correctness of Primev's [standard bridge protocol](https://github.com/primev/mev-commit-bridge/tree/main/standard/bridge-v1) and integration into the mev-commit chain.
 
-Our standard bridge implementation is currently being audited and has always been open source.
+Our standard bridge implementation has been audited by Canita. More details can be found [here](https://blog.primev.xyz/Battle-Testing-mev-commit-Key-Findings-from-Our-Spearbit-Security-Review-1436865efd6f8025bd6ce3824349821d).
```

Comment:
Typo: cantina?

Reply:
Yea that's the same organization as spearbit, will change if it wasn't clear to you